cluster/get-kube-local.sh

```diff
@@ -21,6 +21,7 @@ set -o nounset
 set -o pipefail
 
 KUBE_HOST=${KUBE_HOST:-localhost}
+KUBECONFIG_PATH={KUBECONFIG_PATH:-"/var/run/kubernetes"}
```

Comment:
Good point, changed to using KUBELET_KUBECONFIG and computing dir per below recommendation.